modules/server/certbot/certbot.service.tmpl

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Certbot renewing ssl certificates for https
+After=docker.service,haveged.service,searchbroker.service,auth.service,mdr.service,nginx.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStartPre=/usr/bin/docker exec nginx mkdir -p /tmp/.well-known/acme-challenge
+ExecStartPre=/usr/bin/docker pull certbot/certbot:${version}
+ExecStart=/usr/bin/docker run \
+    --name=certbot \
+    --rm \
+    --network=searchbroker \
+    -v /etc/nginx/ssl:/etc/letsencrypt \
+    -v acme-challenge:/tmp \
+certbot/certbot:${version} certonly \
+    --noninteractive \
+    --agree-tos \
+    --email ${email} \
+    --webroot -w /tmp \
```

Comment:
no /tmp please
why --webroot and -w?

Reply:
It is only temporarly needed to expose the files certbots creates for a few seconds, do you have a directory in mind?

Reply:
--webroot -w specifies the directory will is accesible e.g. at http://search.germanbiobanknode.de/.well-known/acme-challenge/blablabla

Reply:
A now I see. `--webroot` is the module and `-w` is the path. Please use some other dir but not /tmp.